### DISCUSSION TITLE
[Desktop/Hosting] How do I hide the host console window when running Uno Desktop app?

### DISCUSSION THREAD

**Discussion by kucint** [unoplatform/uno#16898] [Q&A] [answered]
Hi,
How am I supposed to hide this window?
![image](https://github.com/unoplatform/uno/assets/54866088/e4e1b7db-66b6-41c9-9900-41f561857af1)

**Comment by jeromelaban** [comment, marked as answer]
Thanks for the report, let's follow up in https://github.com/unoplatform/uno/issues/16905
